src/mapper.rs

```diff
@@ -19,7 +19,7 @@ struct MemberMapping<'s> {
 struct ClassMapping<'s> {
     original: &'s str,
     obfuscated: &'s str,
-    members: BTreeMap<&'s str, Vec<MemberMapping<'s>>>,
+    members: HashMap<&'s str, Vec<MemberMapping<'s>>>,
```

Comment:
This made a small dent in performance as well. Essentially, since there was no reason to keep the members ordered, `HashMap` gave better performance than `BTreeMap` here.